rules-tests/Renaming/Rector/MethodCall/RenameMethodRector/config/configured_rule.php

```diff
@@ -29,7 +29,7 @@
         new MethodCallRename(DifferentInterface::class, 'renameMe', 'toNewVersion'),
 
         // reflection method name
-        new MethodCallRename(ReflectionMethod::class, 'getTentativeReturnType', 'getReturnType'),
+        new MethodCallRename(ReflectionFunctionAbstract::class, 'getTentativeReturnType', 'getReturnType'),
```

Comment:
this for test union: `ReflectionMethod|ReflectionFunction` which both instanceof `ReflectionFunctionAbstract`